sdk/src/main/java/io/opentelemetry/sdk/trace/TracerSdk.java

```diff
@@ -59,6 +60,14 @@
 
   private volatile boolean isStopped = false;
 
+  public TracerSdk() {
+    this(Resource.getEmpty());
+  }
+
+  public TracerSdk(Resource instrumentationLibrary) {
```

Comment:
Why using the same Resource object. I think this will confuse users. What about having a InstrumentationLibraryInfo class with these infos?

Reply:
I was just sticking to the SDK spec which explicitly states a `Resource` has to be created, stored and returned. A POJO/AutoValue with properties would be fine with me as well, I'm not a fan of arbitrary key-value pairs for well-known keys anyway.
Would this require a spec change or is a deviation like that up to the implementation since it doesn't change any functionality?

Reply:
@bogdandrutu what do you say? Should we keep the `Resource` as defined by the OTEP and the SDK spec or modify it and go for a dedicated class?

Reply:
IHMO a dedicated class would be just fine ;)

Reply:
Alright, done!

Reply:
@bogdandrutu @carlosalberto - Alright, done!